src/index.js

```diff
@@ -1 +1,46 @@
 import './style.css';
+import heart from '../assets/heart.png';
+
+const fullPath = 'https://api.tvmaze.com/shows';
+
+// Loading images heart like
+const imgHearts = document.getElementsByClassName('cardboard-heart-image');
+
+for (let i = 0; i < imgHearts.length; i += 1) {
+  imgHearts[i].src = heart;
+}
+
+const getAllShows = async () => {
+  try {
+    const response = await fetch(fullPath);
+    if (!response.status) {
+      throw new Error('Failed to create a post.');
+    }
```

Comment:
**Error Handling:**

In the **getAllShows** function, the condition **if (!response.status)** is not the correct way to check the response status. You should use if **(!response.ok)** to check for errors.